src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeDescriptor.cs

```diff
@@ -3044,6 +3045,10 @@ private sealed class TypeDescriptorComObject
         // string method is the failure is silent during type load making diagnosing the issue difficult.
         private sealed class ComNativeDescriptorProxy : TypeDescriptionProvider
         {
+            [UnsafeAccessor(UnsafeAccessorKind.Constructor)]
+            [return: UnsafeAccessorType("System.Windows.Forms.ComponentModel.Com2Interop.ComNativeDescriptor, System.Windows.Forms")]
+            private static extern object CreateComNativeDescriptor();
```

Comment:
Make it local?

(Our style https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md prefers to have fields the first thing in the type.)